pom.xml

```diff
@@ -94,6 +101,7 @@
 
         <commons-io.version>2.5</commons-io.version>
         <vertx-core.version>3.3.3</vertx-core.version>
+        <fabric8.vertx.plugin.version>1.0-SNAPSHOT</fabric8.vertx.plugin.version>
```

Comment:
makes sense. @rhuss when we do release will the ${project.version} gets updated before pushing to central?